Makefile

```diff
@@ -150,9 +150,29 @@ update: sandstorm-$(BUILD)-fast.tar.xz
 
 fast: sandstorm-$(BUILD)-fast.tar.xz
 
-test: sandstorm-$(BUILD)-fast.tar.xz test-app.spk
+test: sandstorm-$(BUILD)-fast.tar.xz test-app.spk pack-meteor-testapp
 	tests/run-local.sh sandstorm-$(BUILD)-fast.tar.xz test-app.spk
 
+meteor-testapp-clean:
+	rm -rf tests/assets/meteor-testapp.spk meteor-spk-0.3.2 meteor-testapp/.meteor-spk
+
+pack-meteor-testapp:
+	@if [ ! -f ./meteor-spk-0.3.2/meteor-spk ]; then \
```

Comment:
I think it would make sense for this to be a separate make rule for `meteor-spk-0.3.2/meteor-spk`.